CHANGELOG.md

```diff
@@ -1,5 +1,146 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-beta.5
+
+<!-- generated comparing v5.0.0-beta.4..next -->
+
+_Aug 24, 2021_
+
+Big thanks to the 24 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐛 Fixed lots of regressions and bugs to get us closer to the [v5 stable release milestone](https://github.com/mui-org/material-ui/milestone/44)
+- 🗎 Improved the docs and the migration guide for upgrading to v5
```

Comment:
The Emmoji is not supported by Ubuntu. 9% of users use Linux so we should make sure we use a smaller, more widely supported Emoji palette.